### DISCUSSION TITLE
fastf1 on 32-bit python

### DISCUSSION THREAD

**Discussion by nroboka** [theOehrly/Fast-F1#382] [Q&A]
Hi!

I tried to install fastf1 in my 32-bit environment, but the installation failed. So I suspect i cannot use fastf1 with 32 bit python, but if i could that would be awesome, so if anyone does it, please let me know.

**Comment by theOehrly** [comment]
> I tried to install fastf1 in my 32-bit environment, but the installation failed. So I suspect i cannot use fastf1 with 32 bit python, but if i could that would be awesome, so if anyone does it, please let me know.

Hi, am I correct in assuming that you are using Windows as an OS? 

If that's the case, the problem is with Scipy and not with FastF1 directly. FastF1 depends on Scipy for some features and Scipy does no longer provide 32bit Windows builds. There's not really anything I can do about that. The last Scipy build for Windows 32bit was v1.7.3 (only Python 3.9 or older). You could manually install that version of Scipy before installing FastF1 to work around this problem. 

Else, you will need to use a 32bit version of Python on Windows.
